templates/data/summary-complete.html

```diff
@@ -32,47 +32,47 @@
+      <thead><th style="width: 30%;">Voice, timbre, gender, etc.</th><th style="width: 25%;">value</th><th>probability</th></thead>
+      {% for row in summary.other %}
+        <tr><td>{{ row[0] }}</td><td>{{ row[1] }}</td><td>{{ row[2] }}</td></tr>
+      {% endfor %}
+    </table>
 
-      <table class="table table-striped table-bordered">
-        <thead><th style="width: 30%;">Moods</th><th style="width: 25%;">value</th><th>probability</th></thead>
-        {% for row in summary.moods %}
-          <tr><td>{{ row[0] }}</td><td>{{ row[1] }}</td><td>{{ row[2] }}</td></tr>
-        {% endfor %}
-      </table>
+    <table class="table table-striped table-bordered">
+      <thead><th style="width: 30%;">Moods</th><th style="width: 25%;">value</th><th>probability</th></thead>
+      {% for row in summary.moods %}
+        <tr><td>{{ row[0] }}</td><td>{{ row[1] }}</td><td>{{ row[2] }}</td></tr>
+      {% endfor %}
+    </table>
 
-      <table class="table table-striped table-bordered">
-        <thead><th style="width: 30%;">Genres</th><th style="width: 25%;">value</th><th>probability</th></thead>
-        {% for row in summary.genres %}
-          <tr><td>{{ row[0] }}</td><td>{{ row[1] }}</td><td>{{ row[2] }}</td></tr>
-        {% endfor %}
-      </table>
+    <table class="table table-striped table-bordered">
+      <thead><th style="width: 30%;">Genres</th><th style="width: 25%;">value</th><th>probability</th></thead>
+      {% for row in summary.genres %}
+        <tr><td>{{ row[0] }}</td><td>{{ row[1] }}</td><td>{{ row[2] }}</td></tr>
+      {% endfor %}
+    </table>
 
-    {% else %}
-      <div class="alert alert-warning">
-        Sorry, we have not had a chance to calculate the high-level data for
-        this track yet. It should be calculated soon. In the meantime, have a
-        look at our <a href="{{ url_for('data.data') }}#sample-data">sample data</a>.
-      </div>
-    {% endif %}
+  {% else %}
+    <div class="alert alert-warning">
+      Sorry, we have not had a chance to calculate the high-level data for
+      this track yet. It should be calculated soon. In the meantime, have a
+      look at our <a href="{{ url_for('data.data') }}#sample-data">sample data</a>.
+    </div>
+  {% endif %}
 
-    <p>Inspect the data the above is taken from:</p>
-    <ul>
+  <p>Inspect the data the above is taken from:</p>
+  <ul>
+    <li>
+      <a href="{{ url_for('data.view_low_level', mbid=mbid) }}">Low-level data</a>
+      (<a href="{{ url_for('api.get_low_level', mbid=mbid) }}">raw JSON</a>)
+    </li>
+    {% if summary.other -%}
       <li>
-        <a href="{{ url_for('data.view_low_level', mbid=mbid) }}">Low-level data</a>
-        (<a href="{{ url_for('api.get_low_level', mbid=mbid) }}">raw JSON</a>)
+        <a href="{{ url_for('data.view_high_level', mbid=mbid) }}">High-level data</a>
+        (<a href="{{ url_for('api.get_high_level', mbid=mbid) }}">raw JSON</a>)
       </li>
-      {% if other -%}
```

Comment:
It's already fixed. https://github.com/metabrainz/acousticbrainz-server/pull/95/files#diff-9cd37663e642afbcb9d3c3e4c42b952fR71